homework/.keep

```diff
@@ -1 +0,0 @@
-
```

Comment:
Should not push redundant files

Reply:
i just delete this file

Reply:
Thanks @nguyenvanbapc66, if you have time, pls review my pull request #5